src/lib/command.ts

```diff
@@ -135,3 +135,17 @@ export async function sendPermissionsEmbed(msg: Message, error: string) {
         setTimeout(() => permMsg.delete(), PermissionsEmbedDelay); // Delete after 10 seconds
     }
 }
+
+export async function replyToCommandOrQuestion(msg: Message, embed: EmbedBuilder) : Promise<Message<boolean>> {
+    return msg.fetchReference()
+        .then((res) => {
+            let existingFooterText = '';
+            const existingFooter = embed.data.footer;
+            if (existingFooter) {
+                existingFooterText = `${existingFooter.text}\n\n`;
+            }
+            embed.setFooter({ text: `${existingFooterText} Executed by ${msg.author.tag} - ${msg.author.id}` });
```

Comment:
Sorry about previous posts, brainfart. Here's the issue, you update the embed in place, which updates the embed system wide with this footer. So every time this is executed, it adds a new line 

Examples:
<img width="587" alt="Screenshot 2023-03-05 at 11 27 16 PM" src="https://user-images.githubusercontent.com/942391/223045650-cb4f4d46-eef5-4d09-948d-6e41d080f72a.png">
<img width="587" alt="Screenshot 2023-03-05 at 11 27 23 PM" src="https://user-images.githubusercontent.com/942391/223045657-bf78120f-d91d-4cb5-aec1-b820b5cf2700.png">
<img width="599" alt="Screenshot 2023-03-05 at 11 27 41 PM" src="https://user-images.githubusercontent.com/942391/223045658-a28a43c2-3237-41a8-afda-32d595581d1a.png">